Testing/Temporary/LastTest.log

```diff
@@ -0,0 +1,3 @@
+Start testing: Mar 02 00:39 CST
```

Comment:
IDK what these last two testing files are, but if stuff like that gets autogenerated and you **dont** want to push it to github, you can add files to the .gitignore file in base of repo